prql-python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==7.0.0
```

Comment:
FYI I've used poetry very successfully for defining requirements for the past few years (a bit heavier though, so this is maybe better at the moment)

Reply:
I thought about poetry as well! It's my go to - but the reqs list was so small that I thought it was heavier. I'm also not sure how poetry would interact with Maturin - both require build sections, and I don't think pyproject.toml allows for more than one. I'll investigate!